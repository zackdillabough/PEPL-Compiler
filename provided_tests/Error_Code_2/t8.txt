POLY a(M, N) = b^2 - M + N^8736838;
POLY C(P) = x + P - 1;
POLY B(X, Y) = X + Y 
- x^3;
POLY F = x^2 +
 X^2 - 1;
POLY W(U, P) = A
^2 
- A - 1;

POLY Z(M, N) = b^2 - M + N^8736838;

START
INPUT X;
F(X);
INPUT I;
C(I);
273 28 12 3
