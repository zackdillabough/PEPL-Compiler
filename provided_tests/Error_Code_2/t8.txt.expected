Error Code 2: 1 2 4 6 7 9 11 